command/maintenance_window_update.go

```diff
@@ -23,5 +33,38 @@ func (c *MaintenanceWindowUpdate) Synopsis() string {
 }
 
 func (c *MaintenanceWindowUpdate) Run(args []string) int {
+	flags := c.Meta.FlagSet("maintenance-window update")
+	flags.Usage = func() { fmt.Println(c.Help())}
+	if err := flags.Parse(args); err != nil {
+		log.Error(err)
+		return -1
+	}
+	if err := c.Meta.Setup(); err != nil {
+		log.Error(err)
+		return -1
+	}
+	client := c.Meta.Client()
+	var mw pagerduty.MaintenanceWindow
+	if len(flags.Args()) != 1 {
+		log.Error("Please specify input json file")
+		return -1
+	}
+	log.Info("Input file is:", flags.Arg(0))
+	f, err := os.Open(flags.Arg(0))
+	if err != nil {
+		log.Error(err)
+		return -1
+	}
+	defer f.Close()
+	decoder := json.NewDecoder(f)
+	if err := decoder.Decode(&mw); err != nil {
```

Comment:
Since this is a single JSON document and not a stream of them, could we switch this to use `ioutil.ReadFile` + `json.Unmarshal`? There are certain cases where the `json.Decoder` fails to detect malformed JSON, so there are reasons to avoid it unless you're streaming multiple JSON documents.